### DISCUSSION TITLE
table组件，默认选中第一行

### DISCUSSION THREAD

**Discussion by jhyy10** [vbenjs/vue-vben-admin#2485] [Show and tell]
![image](https://user-images.githubusercontent.com/86869776/210470624-9bcb3210-6b89-4039-a706-eab7cb298b73.png)
如图，我的需求是table需要选中某一行进行业务逻辑处理，当table初始化完成之后，或者条件筛选之后，默认选中table的第一行数据。

**Comment by GauharChan** [comment]
useTable有提供`setSelectedRowKeys `方法
